### ISSUE TITLE
Allow just local communications

### ISSUE THREAD

**Issue by RyuzakiKK** [keans/lmnotify#7]
The current local interaction method still requires a first setup using the cloud.
I was hoping to see something like this https://github.com/RyuzakiKK/LaMetric-Apps/blob/80086c7d01b73424f284317a968713808a7a9891/lametric.py#L70 where we could just set the IP address and the api_key, keeping locally the entire communication.

Also in this way the creation of a custom application will not be required anymore.

**Comment by keans** [comment]
Yes, I see your point. I am quite busy at the moment, but I am planning a restructuring of the code and then, it will become easier to implement an "only-local version", which definitely makes sense. However, it will take some time..
